codex-rs/core/src/config.rs

```diff
@@ -142,6 +143,15 @@ pub struct Config {
     /// Definition for MCP servers that Codex can reach out to for tool calls.
     pub mcp_servers: HashMap<String, McpServerConfig>,
 
+    /// Preferred store for MCP OAuth credentials.
+    /// keyring: Use an OS-specific keyring service.
```

Comment:
This implies we are guaranteed to support only one keyring service per OS. Does that feel future-proof?